projects/batfish/src/test/resources/org/batfish/grammar/cisco/testrigs/ibgp-only-reject-routerid-match/configs/r1

```diff
@@ -0,0 +1,18 @@
+! boot system flash:EOS.swi
+hostname r1
+interface Ethernet1
+   no switchport
+   ip address 10.12.0.1/24
+!
+interface Loopback0
+   ip address 1.0.0.1/32
+!
+ip routing
+!
+router bgp 1
+!!! This router ID is the same as r3
+   router-id 9.9.9.9
+   neighbor 10.12.0.2 remote-as 2
```

Comment:
Please peer on loopbacks for iBGP. Use static routes for each loopback that must be reached as a neighbor address.